infra/gcloud-new-akka-node.sh

```diff
@@ -21,7 +21,18 @@ gcloud compute --project "akka-gcp" \
   --image-project "ubuntu-os-cloud" \
   --tags "akka","http-server","https-server" \
   --boot-disk-size "20" --boot-disk-type "pd-ssd" \
-  --boot-disk-device-name "$NAME"
+  --boot-disk-device-name "$NAME" \
+  --metadata startup-script='#!/bin/bash
+    # Each time this node starts, it should attempt running the startup-script provided
+    # Chef scripts prepare the individual scripts, we just make sure we run them on boot
+    
+    cd /home/akka
+    echo "Running all startup-scripts in $(pwd)..." 
+    for script in $(ls *startup-script*); do
+      echo "Running $script"
+      #./$script
+    done
+  '
```

Comment:
this crazy thing allows us to add various startup scripts in each of the independent recipes, and they will all be executed. We need this to allow multiple cookbooks to add their own tweaks. So "technically" though not in practice yet the same node could have the akka tweaks as well as the cassandra tweaks.

This matters if at some point we decide to have more scripts "global" or whatever "io tweaking" etc.